diff --git a/node_modules/@reach/menu-button/dist/menu-button.esm.js b/node_modules/@reach/menu-button/dist/menu-button.esm.js
index 6526e01..60e6a0e 100644
--- a/node_modules/@reach/menu-button/dist/menu-button.esm.js
+++ b/node_modules/@reach/menu-button/dist/menu-button.esm.js
@@ -737,23 +737,19 @@ var MenuPopover = /*#__PURE__*/forwardRef(function MenuPopover(_ref8, forwardedR
   var ref = useForkedRef(popoverRef, forwardedRef);
   useEffect(function () {
     function listener(event) {
-      if (buttonClickedRef.current) {
-        buttonClickedRef.current = false;
-      } else {
-        var relatedTarget = event.relatedTarget,
-            target = event.target; // We on want to close only if focus rests outside the menu
+      var relatedTarget = event.relatedTarget,
+          target = event.target; // We on want to close only if focus rests outside the menu

-        if (isOpen && popoverRef.current) {
-          var _popoverRef$current;
+      if (isOpen && popoverRef.current) {
+        var _popoverRef$current;

-          if (!((_popoverRef$current = popoverRef.current) === null || _popoverRef$current === void 0 ? void 0 : _popoverRef$current.contains(relatedTarget || target))) {
-            dispatch({
-              type: CLOSE_MENU,
-              payload: {
-                buttonRef: buttonRef
-              }
-            });
-          }
+        if (!((_popoverRef$current = popoverRef.current) === null || _popoverRef$current === void 0 ? void 0 : _popoverRef$current.contains(relatedTarget || target))) {
+          dispatch({
+            type: CLOSE_MENU,
+            payload: {
+              buttonRef: buttonRef
+            }
+          });
         }
       }
     }
