diff --git a/node_modules/chart.js/src/scales/scale.time.js b/node_modules/chart.js/src/scales/scale.time.js
index 2496a48..b96ccf5 100644
--- a/node_modules/chart.js/src/scales/scale.time.js
+++ b/node_modules/chart.js/src/scales/scale.time.js
@@ -692,11 +692,16 @@ module.exports = function() {
 		},
 
 		convertTicksToLabels: function(ticks) {
+			var offset = this.options.time.offset;
 			var labels = [];
 			var i, ilen;
 
 			for (i = 0, ilen = ticks.length; i < ilen; ++i) {
-				labels.push(this.tickFormatFunction(moment(ticks[i].value), i, ticks));
+				if (offset) {
+					labels.push(this.tickFormatFunction(moment(moment(ticks[i].value).utc()).utcOffset(offset), i, ticks));
+				} else {
+					labels.push(this.tickFormatFunction(moment(ticks[i].value), i, ticks));
+				}
 			}
 
 			return labels;
