diff --git a/node_modules/wdio-jasmine-framework/build/adapter.js b/node_modules/wdio-jasmine-framework/build/adapter.js
index 399da8a..399ce33 100644
--- a/node_modules/wdio-jasmine-framework/build/adapter.js
+++ b/node_modules/wdio-jasmine-framework/build/adapter.js
@@ -71,6 +71,15 @@ var DEFAULT_TIMEOUT_INTERVAL = 60000;
  * Jasmine 2.x runner
  */
 
+class HookError extends Error {
+    constructor(message) {
+      super(message);
+      this.name = 'HookError';
+    }
+}
+
+var _hookException = null;
+
 var JasmineAdapter = function () {
     function JasmineAdapter(cid, config) {
         var specs = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : [];
@@ -162,6 +171,7 @@ var JasmineAdapter = function () {
                                             return specFn.call(this).then(function () {
                                                 return done();
                                             }, function (e) {
+                                                _hookException = new HookError(e.stack);
                                                 return done.fail(e);
                                             });
                                         };
@@ -242,6 +252,10 @@ var JasmineAdapter = function () {
                             case 25:
                                 result = _context.sent;
                                 _context.next = 28;
+                                if (_hookException != null) {
+                                    console.error(_hookException);
+                                    _hookException = null;
+                                }
                                 return _get__('executeHooksWithArgs')(this.config.after, [result, this.capabilities, this.specs]);
 
                             case 28:
